calico/felix/fiptables.py

```diff
@@ -57,25 +59,26 @@ def on_endpoint_chains_ready(self, iface_name, endpoint_id):
         :param iface_name: name of the linux interface.
         :param endpoint_id: ID of the endpoint, used to form the chain names.
         """
-        if self.iface_to_ep_id_suffix.get(iface_name) != endpoint_id:
-            self.iface_to_ep_id_suffix[iface_name] = endpoint_id
+        if self.iface_to_ep_id.get(iface_name) != endpoint_id:
```

Comment:
Yes - good spot. It was that kind of thing that meant that I didn't want to just commit it! I just fixed that.

It's quite possible that we don't need the map. I wasn't all that sure what it was actually used for, though I was wondering if we would want to know about it, if only because of logging the endpoint id, so I decided to just leave it in for now.